src/atoms/Text/Text.stories.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Text from './Text';
```

Comment:
Move relative imports after package imports. You can also add an eslint rule for that.